src/main/scala/com/datastax/sparkstress/StreamingTask.scala

```diff
@@ -42,7 +37,7 @@ abstract class StreamingTask[rowType](
   }
 
   def setupCQL() = {
-    CassandraConnector(sc.getConf).withSessionDo { session =>
+    CassandraConnector(ss.sparkContext.getConf).withSessionDo { session =>
```

Comment:
Note more to my self, we should make a version of this method which takes SparkSession objects :/